_data/publications/2019.yml

```diff
@@ -7,3 +7,7 @@ publications:
   conference: AAAI Conference on Artificial Intelligence
   pdf: https://arxiv.org/abs/1811.06626
 
+- id: jacobsen2019metadescent
```

Comment:
@ajjacobs @mkschleg @camlinke do any you happen to have a PDF link for this? A quick google search didn't turn one up (though Adam's website had a broken link to it that got my hopes up :stuck_out_tongue: )